.github/ISSUE_TEMPLATE/New_release.md

```diff
@@ -14,36 +14,34 @@ This issue is to provide visibility on the progress of the release process of Gu
 ## Resources
 
 -   📖 Read: [Gutenberg Release Process](https://developer.wordpress.org/block-editor/contributors/code/release/)
--   📖 Read: [Leading a Gutenberg Plugin Release](https://codep2.wordpress.com/2021/07/22/leading-a-gutenberg-plugin-release/)
 -   📽 Watch: [Gutenberg Plugin: New Release Workflow](https://www.youtube.com/watch?v=TnSgJd3zpJY)
 -   📽 Watch: [Creating the Gutenberg plugin v12.0 Release Candidate](https://www.youtube.com/watch?v=FLkLHKecxWg)
 -   📽 Watch: [Gutenberg plugin v12.0.0 Release Party!](https://www.youtube.com/watch?v=4SDtpVPDsLc)
 
 ## Checklist
 
-### RC Day - Wednesday, March 15
+### RC Day - {Weekday, Month, Date}
 
 -   [ ] _Optional:_ Attend `#core-editor` meeting (14:00UTC)
 -   [ ] Post a message in `#core-editor` channel to let folks know you are starting the RC release process
 -   [ ] Organize and Label PRs on the relevant milestone
--   [ ] Start the release process by triggering the `rc` [worklow](https://developer.wordpress.org/block-editor/contributors/code/release/#running-workflow)
+-   [ ] Start the release process by triggering the `rc` [workflow](https://developer.wordpress.org/block-editor/contributors/code/release/#running-workflow)
 -   [ ] [Update the created Draft Release accordingly](https://developer.wordpress.org/block-editor/contributors/code/release/#view-the-release-draft)
 -   [ ] Publish Release
 -   [ ] Announce in `#core-editor` channel that RC1 has been released and is ready for testing
 -   [ ] Ping any other relevant channels announcing that the RC is available
--   [ ] Create Draft of Release post on Make Core blog _(initial draft in [Google doc](https://docs.google.com/document/d/1Hh25EYXSvRd5K45gq0VFN7yfN5l9om2FpX-_KqpVW7g/edit#))_
+-   [ ] Create Draft of Release post on Make Core blog _(initial draft in [Google doc](https://docs.google.com/document/d/1D-MTOCmL9eMlP9TDTXqlzuKVOg_ghCPm9_whHFViqMk/edit))_
 
 ### Between RC and Release
 
--   [ ] [Backport to RC](https://github.com/WordPress/gutenberg/pulls?q=is%3Apr+label%3A%22Backport+to+Gutenberg+RC%22+is%3Aclosed)
--   [ ] [Draft Release Post Highlights and Change Log](https://docs.google.com/document/d/1Hh25EYXSvRd5K45gq0VFN7yfN5l9om2FpX-_KqpVW7g/edit#)
+-   [ ] [Draft Release Post Highlights and Change Log](https://docs.google.com/document/d/1D-MTOCmL9eMlP9TDTXqlzuKVOg_ghCPm9_whHFViqMk/edit)
 -   [ ] Get assets from [Design Team](https://make.wordpress.org/design/) for the post
--   [ ] Reach out to Highlight Authors to draft sections _(if necessary)_
```

Comment:
Not sure why you're dropping this too? Reaching out to authors of highlighted items to get them draft that part of the post could be useful, esp. if the item is particularly technical, or the release lead is less technical.

Reply:
We've never done this in any release that either myself or @justintadlock have done. It seemed extraneous.